src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -63,8 +63,14 @@ public class GroupingsRestControllerv2_1 {
     @Autowired
     private MembershipService membershipService;
 
+    /**
+     * @deprecated Use CURRENT_USER_KEY instead
+     */
+    @Deprecated
```

Comment:
Seems like we still have other places where we use the CURRENT_KEY not just in the files in this PR.

Reply:
FYI... I told him to do that so that we could sorta advertise the name change. Someone should create a Jira to do this change all over.

Reply:
Ok, thank you for letting me know. I just was unsure about the @deprecated annotation because there isn't a clear comment on why we deprecated it for the new key.